test_config.yaml

```diff
@@ -0,0 +1,12 @@
+database:
+  database: cesium_test
+  user: cesium
+  host: localhost
+  port: 5432
+
+server:
+  url: http://localhost:5000
```

Comment:
Neat, we can now run tests on a different port, so that it doesn't conflict with any existing running version!